lambda/replace_accented_chars.js

```diff
@@ -1,77 +1,5 @@
-const replacementMapping = {
-  à: 'a',
-  ê: 'e',
-  ã: 'a',
-  â: 'a',
-  ç: 'c',
-  í: 'i',
-  î: 'i',
-  ú: 'u',
-  û: 'u',
-  õ: 'o',
-  ô: 'o',
-  ó: 'o',
-  Á: 'A',
-  Â: 'A',
-  Ã: 'A',
-  À: 'A',
-  Ç: 'C',
-  È: 'E',
-  Ê: 'E',
-  Í: 'I',
-  Î: 'I',
-  Ì: 'I',
-  Ó: 'O',
-  Ô: 'O',
-  Ò: 'O',
-  Õ: 'O',
-  Ú: 'U',
-  Ù: 'U',
-  Û: 'U',
-  '’': "'",
-  '‘': "'",
-  '“': '"',
-  '”': '"',
-  '–': '-',
-  '\xa0': ' ',
-  '\t': ' ',
-  Δ: '',
-  '¡': '',
-  '¿': '',
-  '£': '',
-  Φ: '',
-  '¥': '',
-  è: 'e',
-  Λ: '',
-  '¤': '',
-  é: 'e',
-  Ω: '',
-  ù: 'u',
-  Π: '',
-  ì: 'i',
-  Ψ: '',
-  ò: 'o',
-  Σ: '',
-  Θ: 'O',
-  Ξ: '',
-  Ø: 'O',
-  Ä: 'A',
-  ä: 'a',
-  ø: 'o',
-  Æ: 'E',
-  Ö: 'O',
-  ö: 'o',
-  æ: 'e',
-  Ñ: 'N',
-  ñ: 'n',
-  Å: 'A',
-  ß: '',
-  Ü: 'U',
-  ü: 'u',
-  å: 'a',
-  É: 'E',
-  '§': ''
-};
+const { getOpts } = require('./utils');
+const replacementMapping = require('../src/static/UCS-2-substitutes.json');
```

Comment:
I think it's better to name it `GSM7_REPLACEMENTS` like we have in our other projects.
https://github.com/greatnonprofits-nfp/rapidpro/blob/aa991c4e2e02599e3bb9c47e60e5265038c91fd3/temba/utils/gsm7.py#L19
https://github.com/nyaruka/gocommon/blob/bdb0c2865ae7161dc540adea53cb21e37fdcfbbb/gsm7/gsm7.go#L152